junifer/data/parcellations/_ants_parcellation_warper.py

```diff
@@ -131,7 +131,7 @@ def warp(
             # Get template space image
             template_space_img = get_template(
                 space=dst,
-                target_data=target_data,
+                target_img=parcellation_img,
```

Comment:
i think this one needs to be in the refactor of the parcellation logic, otherwise it will fail tests here.

Reply:
I updated it for both parcellation and mask to reduce the confusion in the other PRs.